Content.Goobstation.Server/HisGrace/HisGraceSystem.cs

```diff
@@ -158,8 +159,8 @@ private void OnEntityConsumed(Entity<HisGraceComponent> hisGrace, ref HisGraceEn
         if (!TryComp<MeleeWeaponComponent>(hisGrace, out var melee))
             return;
 
-        // 5 blunt per entity consumed
-        hisGrace.Comp.CurrentDamageIncrease.DamageDict["Blunt"] = hisGrace.Comp.EntitiesAbsorbed * 5;
+        // 5 blunt per entity consumed > now 1.4, it was scaling too fast and trivialized ascending
```

Comment:
why doesnt this idiot just change the comment to say what it is instead of play pretending CLs
```suggestion
        // 1.4 blunt per entity consumed